doc/benchmark_workflow/manage_dependencies.rst

```diff
@@ -0,0 +1,55 @@
+.. _manage_dependencies:
+
+Manage dependencies
```

Comment:
maybe call this "Install a benchmark"?
We will add later some info on how to install the benchmark and on conda env management